dht.go

```diff
@@ -111,8 +114,8 @@ func New(ctx context.Context, h host.Host, options ...opts.Option) (*IpfsDHT, er
 // NewDHT creates a new DHT object with the given peer as the 'local' host.
 // IpfsDHT's initialized with this function will respond to DHT requests,
 // whereas IpfsDHT's initialized with NewDHTClient will not.
-func NewDHT(ctx context.Context, h host.Host, dstore ds.Batching) *IpfsDHT {
-	dht, err := New(ctx, h, opts.Datastore(dstore))
+func NewDHT(ctx context.Context, h host.Host, dstore ds.Batching, bucketSize int) *IpfsDHT {
```

Comment:
There's no need to break this. We can just use `New`.

Reply:
True. I took the original patch and applied it. I'll remove this hunk.